CODEOWNERS

```diff
@@ -148,6 +148,7 @@ homeassistant/components/gios/* @bieniu
 homeassistant/components/gitter/* @fabaff
 homeassistant/components/glances/* @fabaff @engrbm87
 homeassistant/components/gogogate2/* @vangorra
+homeassistant/components/google/* @MartinHjelmare @wernerhp
```

Comment:
Does @MartinHjelmare know he is being added?

Reply:
:laughing: Please remove me. Thanks! :+1: